### ISSUE TITLE
librarian-chef install rejects the "R" cookbook

### ISSUE THREAD

**Issue by kgilpin** [applicationsonline/librarian#89]
Cookbook name "R" is rejected as "not sensible"

http://community.opscode.com/cookbooks/R

```
raise ArgumentError, "name (#{name.inspect}) must be sensible" unless name =~ /^\S.*\S$/
```

**Comment by yfeldblum** [comment]
Ha!

**Comment by yfeldblum** [comment]
Fixed by 7325ce332eddbbf007ee9ee226fd8240fda1bb77.
